### ISSUE TITLE
Height datapack doesn't work out of the box.

### ISSUE THREAD

**Issue by uwainium** [BTE-Germany/TerraPlusMinus#28]
Because of the update 1.20.5 which changes the way int providers are handled, the datapack doesn't work anymore. 
![image](https://github.com/user-attachments/assets/527e3f43-add6-4362-9312-cae8434c9785)

**Comment by RocketStrike24** [comment]
do you know how to fix it

**Comment by uwainium** [comment]
Yeah I already made the fix locally so our server would work, but I wanted to make a ticket so that it would be updated here as well.

**Comment by Zoriot** [comment]
> Yeah I already made the fix locally so our server would work, but I wanted to make a ticket so that it would be updated here as well.

Can you make a PR with the fixed pack?
Using another also works. Like [2032 World Hight](https://modrinth.com/datapack/2032-world-height)
